tests/conftest.py

```diff
@@ -106,9 +106,15 @@ def experiment_dir_merged(experiment_dir, active_config):
 
     current_dir = os.getcwd()
     ensure_constraints_file_presence(current_dir)
-    destination = assemble_experiment_temp_dir(active_config)
-    os.chdir(destination)
-    yield
+    with mock.patch(
+        "dallinger.deployment.get_editable_dallinger_path"
+    ) as get_editable_dallinger_path:
+        # When dallinger is not installed as editable egg the requirements
+        # file sent to heroku will include a version pin
+        get_editable_dallinger_path.return_value = False
```

Comment:
This patch seems sensible, but what caught my eye was the return value of the patched function. Can we make that return `None` instead? `str` or `False` really confused me!

Reply:
Yep, it's confusing. I leaned towards a boolean because in the end it has to be a falsy value. But even an empty string would have worked, maintaining a single type.

Fixed.